### ISSUE TITLE
is it no need to think the conformance level A?

### ISSUE THREAD

**Issue by Junko-Koide** [w3c/silver#325]
@MakotoUeki -san, would you please translate from Japanese to English? 

Bronzeが適合レベルAA相当というのは、求められるレベルとしては高いと感じました。
一方、ユーザーがハンディキャップによらずサイトを利用するために求められる最低限のレベルとしてはAAは妥当と思っています。また、W3C/WCAGとして、ユーザーがサイトをある程度利用できるもっとも低い目標値としてAA相当をBronzeと打ち出す、という理想を明確にすることも必要と思います。

EU/米国のように、法律とあわせてサイトのアクセシビリティ対応をすすめられる国・地域ばかりではないという現実があります。
理想に現実をあわせていける国や地域はそのまま他国の目標となるべく、どんどんすすめていただくのがよいと思いますが、これから適合レベルAの内容のアクセシビリティ対応が最短の目標となる国や地域にとっては、W3C/WCAGというグローバルレベルの水準に届くための目標が消えることになります。
アクセシビリティ対応が不十分な国や地域にとってこそ、グローバルが目標として求めているという事実が、アクセシビリティ対応を促進するための後押しのひとつとして必要なのではないでしょうか。
どの国・地域のユーザーでも情報を得られるようになることは非常に重要だと思います。

適合レベルAA相当をBronzeとして最低限満たす目標とするのは賛成ですが、適合レベルA相当がBronzeへステップアップするための足がかりはまだまだ現実的に必要な状況だと思います。
高い目標が実現できる国・地域とできない国・地域との二極化は避けるべきです。それはインターネットの理念にそぐわないと思います。

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by lauracarlson** [comment]
Copied from translation tool:

“I felt that Bronze's conformity level AA was high as a required level.
On the other hand, I think AA is reasonable as the minimum level required for users to use the site regardless of handicap. Also, as W3C / WCAG, I think it is necessary to clarify the ideal that AA equivalent is set as Bronze as the lowest target value that users can use the site to some extent.

The reality is that not all countries / regions, such as the EU / US, are encouraged to support site accessibility in accordance with the law.
I think it would be better to continue to promote countries and regions that can match the reality to their ideals as they are, so that they can be the goals of other countries. , The goal of reaching the global level of W3C / WCAG will disappear.
For countries and regions with inadequate accessibility, the fact that the global goal is required may be necessary as one of the boosts to promote accessibility.
I think it is very important for users in any country / region to be able to obtain information.

I agree with the goal of satisfying the minimum conformance level AA equivalent as Bronze, but I think that a foothold for the conformance level A equivalent to step up to Bronze is still practically necessary.
We should avoid the polarization of countries / regions that can achieve high goals and those that cannot. I don't think it fits the idea of ​​the Internet."

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
